### PR TITLE
Upgrade to postgres 13.3

### PR DESCRIPTION
**Pull request by ewdurbin**

### CODE REVIEW

File: tests/unit/legacy/api/test_json.py [hunk 1]
```diff
@@ -249,7 +247,7 @@ def test_detail_renders(self, pyramid_config, db_request, db_session):
         expected_urls = []
         for project_url in reversed(project_urls):
             expected_urls.append(tuple(project_url.split(",")))
-        expected_urls = OrderedDict(tuple(expected_urls))
+        expected_urls = dict(tuple(expected_urls))
```
Comment (di) [inline tests/unit/legacy/api/test_json.py:250]: I think the order matters here, did we lose the implicit ordering in the DB?

Comment (ewdurbin) [reply tests/unit/legacy/api/test_json.py]: seems we still have implicit ordering... but it's not reversed? see a112d01b

File: tests/unit/legacy/api/test_json.py [hunk 2]
```diff
@@ -245,9 +247,9 @@ def test_detail_renders(self, pyramid_config, db_request, db_session):
             r"unsafechars,http://example.com <>[]{}|\^%",
         ]
         expected_urls = []
-        for project_url in reversed(project_urls):
+        for project_url in project_urls:
```
[the remark this answers is not in the document]
Comment (ewdurbin) [reply tests/unit/legacy/api/test_json.py:250]: It's not clear to me how the order of URLs after "Homepage" and "Download" strictly matter: https://github.com/pypa/warehouse/blob/3e18616b1d22b5399af493c9e2fcfc11df74b61b/warehouse/packaging/models.py#L467-L483

[the remark this answers is not in the document]
Comment (ewdurbin) [reply tests/unit/legacy/api/test_json.py:249]: updated the test to actually exercise the homepage part, see 702447701bf2b160fbdc62900a7266b0a866603e

**Review by di — APPROVED** [review]

**Comment by miketheman** [comment]
Did prod ever get the Postgres version upgrade to 13, or possibly even 14?

**Comment by ewdurbin** [comment]
Closing, we'll go to 14.4 :)

**Pull request closed without merging**